### ISSUE TITLE
Error of Sentinel-5P data request

### ISSUE THREAD

**Issue by IBunDem** [sentinel-hub/sentinelhub-py#94]
```
Python 3.7.3.
>> pip freeze
aenum==2.2.3
boto3==1.10.47
botocore==1.13.47
certifi==2019.11.28
chardet==3.0.4
Click==7.0
cycler==0.10.0
docutils==0.15.2
idna==2.8
imagecodecs-lite==2019.12.3
jmespath==0.9.4
kiwisolver==1.1.0
matplotlib==3.1.2
numpy==1.18.1
Pillow==7.0.0
pyparsing==2.4.6
pyproj==2.4.2.post1
python-dateutil==2.8.1
requests==2.22.0
s3transfer==0.2.1
sentinelhub==2.6.1
Shapely==1.6.4.post2
six==1.13.0
tifffile==2019.7.26.2
urllib3==1.25.7
utm==0.5.0
```
I tried to create default SEntinel-5P template and custom empty template with 1 layer using SENTINEL-5P data.
INSTANCE_ID is correct. in layer selected Source Sentinel-5P - L2. Try make request like this:
```
    conf_id = '%Here my instance id of default sentinel-hub SENTINEL-5P template%'
    layer = 'AER_AI_340_380'
    coords = [46.16, -16.15, 46.51, -15.58]
    bbox = BBox(bbox=coords, crs=CRS.WGS84)
    request = WcsRequest(data_folder='test_dir',
                                   data_source=DataSource.SENTINEL5P,
                                   layer=layer,
                                   bbox=bbox,
                                   time=('2018-12-15', '2020-01-08'),
                                   image_format=MimeType.TIFF_d32f,
                                   instance_id=conf_id)
    img = request.get_data(save_data=True)
```

But I get an Error
```
/home/mishka/Documents/python/venvs/sentinelhub/lib/python3.7/site-packages/pyproj/crs.py:77: FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method.
  return _prepare_from_string(" ".join(pjargs))
/home/mishka/Documents/python/venvs/sentinelhub/lib/python3.7/site-packages/pyproj/crs.py:77: FutureWarning: '+init=<authority>:<code>' syntax is deprecated. '<authority>:<code>' is the preferred initialization method.
  return _prepare_from_string(" ".join(pjargs))
Traceback (most recent call last):
  File "/home/mishka/Documents/python/venvs/sentinelhub/lib/python3.7/site-packages/sentinelhub/download.py", line 236, in execute_download_request
    response.raise_for_status()
  File "/home/mishka/Documents/python/venvs/sentinelhub/lib/python3.7/site-packages/requests/models.py", line 940, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://services.sentinel-hub.com/ogc/wfs/2c14d0df-74cd-477e-88e1-da5e711c6c48?SERVICE=wfs&REQUEST=GetFeature&TYPENAMES=S5p_L2.TILE&BBOX=-16.15%2C46.16%2C-15.58%2C46.51&OUTPUTFORMAT=application%2Fjson&SRSNAME=EPSG%3A4326&TIME=2018-12-15T00%3A00%3A00%2F2020-01-08T23%3A59%3A59&MAXCC=100.0&MAXFEATURES=100&FEATURE_OFFSET=0

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mishka/Documents/python/venvs/sentinelhub/lib/python3.7/site-packages/sentinelhub/download.py", line 257, in execute_download_request
    raise DownloadFailedException(_create_download_failed_message(exception, request.url))
sentinelhub.download.DownloadFailedException: Failed to download from:
https://services.sentinel-hub.com/ogc/wfs/2c14d0df-74cd-477e-88e1-da5e711c6c48?SERVICE=wfs&REQUEST=GetFeature&TYPENAMES=S5p_L2.TILE&BBOX=-16.15%2C46.16%2C-15.58%2C46.51&OUTPUTFORMAT=application%2Fjson&SRSNAME=EPSG%3A4326&TIME=2018-12-15T00%3A00%3A00%2F2020-01-08T23%3A59%3A59&MAXCC=100.0&MAXFEATURES=100&FEATURE_OFFSET=0
with HTTPError:
400 Client Error: Bad Request for url: https://services.sentinel-hub.com/ogc/wfs/2c14d0df-74cd-477e-88e1-da5e711c6c48?SERVICE=wfs&REQUEST=GetFeature&TYPENAMES=S5p_L2.TILE&BBOX=-16.15%2C46.16%2C-15.58%2C46.51&OUTPUTFORMAT=application%2Fjson&SRSNAME=EPSG%3A4326&TIME=2018-12-15T00%3A00%3A00%2F2020-01-08T23%3A59%3A59&MAXCC=100.0&MAXFEATURES=100&FEATURE_OFFSET=0
Server response: "TYPENAME=S5p_L2.TILE is invalid!"
```

**Comment by IBunDem** [comment]
OS Kubuntu 19.04 x64. Also tried on MacOS

**Comment by IBunDem** [comment]
In real here is 2 errors after warrining about depricated syntax of pyproj module.
```
Traceback (most recent call last):
  File "/home/mishka/Documents/python/venvs/sentinelhub/lib/python3.7/site-packages/sentinelhub/download.py", line 236, in execute_download_request
    response.raise_for_status()
  File "/home/mishka/Documents/python/venvs/sentinelhub/lib/python3.7/site-packages/requests/models.py", line 940, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://services.sentinel-hub.com/ogc/wfs/2c14d0df-74cd-477e-88e1-da5e711c6c48?SERVICE=wfs&REQUEST=GetFeature&TYPENAMES=S5p_L2.TILE&BBOX=-16.15%2C46.16%2C-15.58%2C46.51&OUTPUTFORMAT=application%2Fjson&SRSNAME=EPSG%3A4326&TIME=2018-12-15T00%3A00%3A00%2F2020-01-08T23%3A59%3A59&MAXCC=100.0&MAXFEATURES=100&FEATURE_OFFSET=0
```
```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mishka/Documents/python/venvs/sentinelhub/lib/python3.7/site-packages/sentinelhub/download.py", line 257, in execute_download_request
    raise DownloadFailedException(_create_download_failed_message(exception, request.url))
sentinelhub.download.DownloadFailedException: Failed to download from:
https://services.sentinel-hub.com/ogc/wfs/2c14d0df-74cd-477e-88e1-da5e711c6c48?SERVICE=wfs&REQUEST=GetFeature&TYPENAMES=S5p_L2.TILE&BBOX=-16.15%2C46.16%2C-15.58%2C46.51&OUTPUTFORMAT=application%2Fjson&SRSNAME=EPSG%3A4326&TIME=2018-12-15T00%3A00%3A00%2F2020-01-08T23%3A59%3A59&MAXCC=100.0&MAXFEATURES=100&FEATURE_OFFSET=0
with HTTPError:
400 Client Error: Bad Request for url: https://services.sentinel-hub.com/ogc/wfs/2c14d0df-74cd-477e-88e1-da5e711c6c48?SERVICE=wfs&REQUEST=GetFeature&TYPENAMES=S5p_L2.TILE&BBOX=-16.15%2C46.16%2C-15.58%2C46.51&OUTPUTFORMAT=application%2Fjson&SRSNAME=EPSG%3A4326&TIME=2018-12-15T00%3A00%3A00%2F2020-01-08T23%3A59%3A59&MAXCC=100.0&MAXFEATURES=100&FEATURE_OFFSET=0
Server response: "TYPENAME=S5p_L2.TILE is invalid!"
```

I tried not to use data_source parameter in WcsRequest, just say about S2 tile not found.
Other satellites give data without problems.

**Comment by holonomicjl** [comment]
#120

**Comment by IBunDem** [comment]
> 
> 
> Pull request #120 should solve this issue, but you still have to set the sh_base_url configuration parameter.

Thank you for your work and answer! But I still have problems with getting snapshots from S5P.
Now main problem is:
 'Server response: "Dataset type S5PL2 not supported here. Please use https://creodias.sentinel-hub.com/ endpoint." '. I see, Here is incorrect Data source name, but in sentinel-hub documentation is not any about this.

**Comment by holonomicjl** [comment]
config.sh_base_url = "https://creodias.sentinel-hub.com" should be good, but there's one more thing:

Did you pass   data_source=DataSource.SENTINEL5P to the request? I think I had the same problem and that was the soultion.

**Comment by holonomicjl** [comment]
Also you can use logging.basicConfig(level=logging.INFO) and debug the http requests in your browser first using the HTTP API directly.

**Comment by AleksMat** [comment]
PR https://github.com/sentinel-hub/sentinelhub-py/pull/120 has been merged and a new version `3.0.3` released.

Closing the issue.
